BioArchLinux/spades/PKGBUILD

```diff
@@ -12,10 +12,11 @@ depends=(
   python
 )
 makedepends=(
-  bzip2
+  gcc
```

Comment:
gcc is the default compiler, I don't think we need to add it.

Reply:
I used `makepkg` to test building process, then I got some errors.
After installing `gcc`, it works. That's why I added gcc into PKGBUILD.

Reply:
But have you tried bioarchlinux-x86_64-build?

Reply:
`gcc` is a dependency of `base-devel`, so it should not be in makedepends. From [Arch wiki](https://wiki.archlinux.org/title/PKGBUILD#makedepends):

> The package [`base-devel`](https://archlinux.org/packages/?name=base-devel) is assumed to be already installed when building with makepkg. Dependencies of this package **should not** be included in `makedepends` array.

Reply:
Thanks. Removed `gcc`.